src/dropdowns/README.md

```diff
@@ -0,0 +1,19 @@
+# Primer dropdown
+
+> Dropdowns are lightweight, JavaScript-powered context menus for housing navigation and actions. They're great for instances where you don't need the full power (and code) of the select menu.
```

Comment:
Are they js powered? I thought dropdowns full relied on the native `<details>` behavior? I don't see any js classes in the documentation. If so, let's make sure to specify which class is being targeted by the JS so that the user knows it's mandatory to include.

Not sure if you added this or just moved it over, but we should probably update if my assumptions are correct!

Reply:
I guess it used to be "js powered" but now using `<details>` is recommended. 👍 

Note: This `README.md` will be overridden when running `script/generate-bundle-readmes`. I'll remove `, JavaScript-powered` from the [docs](https://github.com/primer/css/blob/fcf410b84d826f402618a2a9338f580e4ae0663f/docs/content/components/dropdown.md#L31) tough.